tests/introspection/Mac/MacApiCtorInitTest.cs

```diff
@@ -43,6 +43,28 @@ protected override bool Skip (Type type)
 			case "MonoMac.QuickLookUI.QLPreviewPanel":
 				return true;
 			// These should be DisableDefaultCtor but can't due to backward compat
+			case "AppKit.NSCollectionViewTransitionLayout":
```

Comment:
Can you open a bug that we should not check obsolete ctor, I think it will ease future maintenance. Thanks!